controllers/teams.js

```diff
@@ -286,19 +286,19 @@ router.get('/', async (req, res, next) => {
 
 	if (req.query.json) {
 		res.json(teams);
-	} else {
+	} else if (teams.length > 0 || teamInvitations.length > 0) {
```

Comment:
!== is faster than > and has clearer meaning

Reply:
thanks for this